test/integration/joint_spawn.cc

```diff
@@ -162,7 +162,34 @@ void JointSpawningTest::SpawnJointTypes(const std::string &_physicsEngine,
     }
   }
 
-  // world as child of a joint is not supported in sdf 1.7
```

Comment:
This had been removed on purpose on https://github.com/osrf/gazebo/commit/e4b30950df5ce8cbc8c5f617660d18065e4939ce